pyproject.toml

```diff
@@ -50,13 +50,17 @@ isort = "^5.13.2"
 
 [tool.poetry.group.test.dependencies]
 pytest-timeout = "^2.3.1"
+pytest-env = "^1.1.5"
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pytest.ini_options]
 log_cli = true
+env = [
+    "HATCHET_CLIENT_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJodHRwOi8vbG9jYWxob3N0OjEyMzQiLCJleHAiOjk5OTk5OTk5OTksImdycGNfYnJvYWRjYXN0X2FkZHJlc3MiOiJodHRwOi8vbG9jYWxob3N0OjQ0MyIsImlhdCI6MTIzNDU2Nzg5MSwiaXNzIjoiaHR0cDovL2xvY2FsaG9zdDoxMjM0Iiwic2VydmVyX3VybCI6Imh0dHA6Ly9sb2NhbGhvc3Q6MTIzNCIsInN1YiI6IjAwMDAwMDAwLTVmN2QtNGM1NS1iZmEzLWFkZDk0MTc4YjhmNyIsInRva2VuX2lkIjoiMDAwMDAwMDAtZmU5ZS00ZGEyLThmOTgtNTQ5YTgxOWRmZTE5In0.bIly53KfKcXP_7wjySWvbmxG9cVqit-fzVQAF5K7rPc",
```

Comment:
@mrkaye97 local token?

Reply:
yeah, I made a fake one, figure it's fine to have here for tests 🤷

Reply:
i see but it will only work for you... right?

Reply:
ah no, sorry - the intention here is to be able to test config loading from env vars, but since we instantiate the `ClientConfig` as a default in a bunch of places (which we shouldn't do, probably) and those files get imported, the env var needs to be set before any imports happen, hence this change. 

We should change this going forward to not have defaults for those I think (or maybe something else - TBD)

Reply:
Gotcha, I'm sure we'll forget and have this conversation in 6 months

Reply:
😂 no doubt about that, hopefully this is gone by then lol